main.js

```diff
@@ -9,6 +9,8 @@ const autoUpdaterInterval = 60 * 60 * 1000;
 const ipc = electron.ipcMain;
 const Menu = electron.Menu;
 const shell = electron.shell;
+const ElectronConfig = require('electron-config');
+const userConfig = new ElectronConfig();
```

Comment:
I haven't tested it but think you might want to move this constructor call to after line 70 or so, where we conditionally change the userData directory based on configs. Otherwise your prod/dev/staging/test env window settings will all be read/written from the same place. Not sure if that's a bug or a feature...